### PR TITLE
[IMP] Round floats to two decimals

### PR DESCRIPTION
**Pull request by RoelAdriaans**
We use a database with multi-currency, and because Odoo's habbit to use floats, we also have floats in the database. 

Report does no rounding, giving us the wonderfull errors:
<pre>
 :140197:0:ERROR:SCHEMASV:SCHEMAV_CVC_FRACTIONDIGITS_VALID: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '1183.03194186' has more fractional digits than are allowed ('2').

:140197:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': '1183.03194186' is not a valid value of the atomic type '{http://www.auditfiles.nl/XAF/3.2}Amount2decimals'.

:140209:0:ERROR:SCHEMASV:SCHEMAV_CVC_FRACTIONDIGITS_VALID: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '1183.03194186' has more fractional digits than are allowed ('2').

:140209:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': '1183.03194186' is not a valid value of the atomic type '{http://www.auditfiles.nl/XAF/3.2}Amount2decimals'.

:143824:0:ERROR:SCHEMASV:SCHEMAV_CVC_FRACTIONDIGITS_VALID: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '1183.03194186' has more fractional digits than are allowed ('2').

:143824:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': '1183.03194186' is not a valid value of the atomic type '{http://www.auditfiles.nl/XAF/3.2}Amount2decimals'.

:143836:0:ERROR:SCHEMASV:SCHEMAV_CVC_FRACTIONDIGITS_VALID: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '1183.03194186' has more fractional digits than are allowed ('2').

:143836:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': '1183.03194186' is not a valid value of the atomic type '{http://www.auditfiles.nl/XAF/3.2}Amount2decimals'.

:147916:0:ERROR:SCHEMASV:SCHEMAV_CVC_FRACTIONDIGITS_VALID: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '591.515970931' has more fractional digits than are allowed ('2').
</pre>

This patch adds rounding to the report.

### CODE REVIEW

File: l10n_nl_xaf_auditfile_export/models/ir_qweb.py [hunk 1]
```diff
@@ -34,6 +35,21 @@ def value_to_html(self, value, options):
...
 
 
+class IrQwebAuditfileFloatWidget2(models.AbstractModel):
+    _name = 'ir.qweb.field.auditfile.round2'
+    _inherit = 'ir.qweb.field'
+    _max_digits = 2
+
+    @api.model
+    def value_to_html(self, value, options):
+        value_in = value
```
Comment (hbrunn) [inline l10n_nl_xaf_auditfile_export/models/ir_qweb.py]: this is unused

**Review by hbrunn — CHANGES_REQUESTED** [review]
that on the way fixes #129 too, right?

**Review by hbrunn — APPROVED** [review]

**Comment by RoelAdriaans** [comment]
@hbrunn Yes, in theory this should also solve  #129 ..

**Comment by astirpe** [comment]
IMO the rounding issue could be fixed in a simpler way, just modifying one single line, and there's not any need to create a new widget.
Just replace line:
```
<amnt><t t-esc="l.credit or l.debit" /></amnt>
```
with:
```
<amnt><t t-esc="abs(l.balance)" /></amnt>
```

I noticed a similar issue while porting the module to V11 and [this fix](https://github.com/onesteinbv/l10n-netherlands/blob/54e0b2beabff895d13156e0f67a93cc4768558e0/l10n_nl_xaf_auditfile_export/views/templates.xml#L150) seems working fine.

**Comment by astirpe** [comment]
This PR could be closed IMO. Superseded by https://github.com/OCA/l10n-netherlands/pull/153

**Pull request closed without merging**